packages/graph-explorer/src/utils/createDisplayError.ts

```diff
@@ -46,6 +46,18 @@ export function createDisplayError(error: any): DisplayError {
       };
     }
 
+    // Malformed query
```

Comment:
a bit off-topic question:
Do we have some kind run-book on how to solve such problems?

Reply:
Not that I'm aware of. I would assume we do our best to ensure the queries run on a wide variety of database versions, and patch any issues that come up.

@michaelnchin Do you have any thoughts on this?